pkg/operator/ceph/object/user.go

```diff
@@ -185,6 +198,11 @@ func DeleteUser(c *Context, id string, opts ...string) (string, error) {
 		if code, ok := exec.ExitStatus(err); ok && code == int(syscall.ENOENT) {
 			return result, nil
 		}
+
+		res, er := ListUserBucket(c, id)
+		if er == nil && res != "" && res != "[]" {
+			return result, errors.Wrap(err, fmt.Sprintf("failed to delete s3 user with buckets %q", res))
```

Comment:
you can use `errors.Wrapf` and remove `fmt.Sprintf`. [For example](https://github.com/rook/rook/blob/f86f8300f924b98e7df37207276ee40a4bd33870/pkg/operator/ceph/object/objectstore.go#L133)

Reply:
Done